ReactNative/ExampleSuperApp/ios/GeneXus Frameworks/README.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
This folder is not needed, point to [iOS/GeneXus Frameworks](https://github.com/genexus-books/gx-super-app/tree/main/iOS/GeneXus%20Frameworks)